apps/utils/locust_tests/v1/regression_suite.py

```diff
@@ -77,17 +78,17 @@ class RegressionV1User(BFDUserBase):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.bene_ids = master_bene_ids.copy()
-        self.contract_data = master_contract_data.copy()
-        self.hashed_mbis = master_hashed_mbis.copy()
+        self.bene_ids = itertools.cycle(master_bene_ids.copy())
```

Comment:
[`itertools.cycle`](https://docs.python.org/3/library/itertools.html#itertools.cycle) returns an iterator on an iterable that, once reaching the end of the iterable, loops back around to the first element in the given iterable. Thus, data is never exhausted when running tests for an arbitrary amount of time.

This is _especially_ important for PAC data, as the number of synthetic MBIs is _very_ small in each environment. We would exhaust the list of MBIs within _seconds_ without this new behavior.